.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": ["next/core-web-vitals", "prettier"],
+  "plugins": ["prettier"],
+  "rules": {
+    "prettier/prettier": "error",
+    "import/order": ["error", { "newlines-between": "always" }],
+    "no-console": ["error", { "allow": ["info", "warn", "error"] }],
+    "spaced-comment": ["error", "always", { "markers": ["/"] }]
+  }
+}
```

Comment:
## Suggestion 
can you also add this rule? we normally also have this in other projects
`"no-unused-vars": "error"`